blog.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
+      <section id="banner">
+          <div class="filler">
+            </div>
+        <div id="letterhead">
+          <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+          Seattle's premiere dog adoption agency
+        </div>
+        <div id="cart">
+          <div class="top-cart">
+            <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo">
+            &nbsp;&nbsp;$1234.56
```

Comment:
instead of pushing these over with non-breaking spaces, I'd rather have margin do it. Just in case a browser reads those funny . Odds are they won't, but it's best to let css do the styling, not force it with conent